File: backend/api/urls.py
from rest_framework.routers import DefaultRouter
from .views import *
from django.urls import path, include

# Create your views here.
router = DefaultRouter()
router.register(r'faculty', FacultyViewSet)
router.register(r'query', QueryViewSet)
router.register(r'topic', TopicViewSet)

urlpatterns = [
    # path(r'trash/', DeletedItemList.as_view()),
    # path(r'warehouse/', WarehouseList.as_view()),
    path(r'faculty/<int:pk>', FacultyDetail.as_view()),
    path(r'users/<int:pk>', FacultyDetail.as_view()),
    path(r'topic/<int:id>', TopicDetail.as_view()),
    path(r'', include(router.urls), name="add items or warehouse"),
]

File: src/urls.ts

import hello from "@api/hello";
import topic from "@api/topic";
import auth from "@api/auth/urls";
import post from "@api/post";
import express from "express";
import faculty from "@api/faculty";
import doc from "@doc/doc";

export default express.Router()
    .use('/api-doc', doc)
    .use('/hello', hello)
    .use('/topic', topic)
    .use('/faculty', faculty)
    .use('/post', post)
    .use('/auth', auth);
    
